go.mod

```diff
@@ -84,6 +87,7 @@ require (
 	github.com/stretchr/testify v1.8.4
 	github.com/tidwall/gjson v1.16.0
 	github.com/tidwall/sjson v1.2.5
+	github.com/urfave/cli v1.22.2
```

Comment:
can we skip v1 and just use v2?

Reply:
I will create a task to address this.

Reply:
v1 is seems to be supported btw https://github.com/urfave/cli/releases/tag/v1.22.14, so it not an urgent security reason to upgrade.